extensions/64-bit-address-space/README.md

```diff
@@ -0,0 +1,58 @@
+# General Design
+
+**Extension State: Under Development**  
+**Requires: CP1.15**  
+**CPUID Bit: CP1.32**
+
+# Overview
+
+This extension adds a new concept of _modes_ to the ISA. Modes are used to control things about the processor
+state which are binary-incompatible - that is, binary code written for a mode other than the mode the
+processor is actually in will almost certainly behave incorrectly.
+
+In particular, this extension focuses on the _Real 64-bit address mode_.
+
+# Modes
+
+CR `1 0001`, equivalently `cr17` or "the MODE control register," holds a value indicating the current
+_mode_ of the processor. This has an initial value of `0`.
```

Comment:
Could we apply Mega's suggestion here of having a separate documentation file to describe the `mode` register and link to each mode-extension from there and vice versa?

Reply:
I've done my best at implementing it. Give it a look.